recipes/qt/5.x.x/conanfile.py

```diff
@@ -802,6 +802,9 @@ def _create_plugin(pluginname, libname, type, requires):
             core_reqs.append("glib::glib-2.0")
 
         _create_module("Core", core_reqs)
+        if self.settings.compiler == "Visual Studio":
+            self.cpp_info.components["qtCore"].exelinkflags.append("-ENTRY:mainCRTStartup")
```

Comment:
Isn't this pushing people using a `main` entry point?
See https://docs.microsoft.com/en-us/cpp/build/reference/entry-entry-point-symbol?view=msvc-160

It should still be possible to use the `WinMainCRTStartup` entry point with qt.

Reply:
I think you can still override the flag downstream. This change just makes the package behave by default like regular qt

Reply:
I'm no qt user so I won't complain.

Reply:
I am a qt user and I have this issue. I'm currently trying to migrate from a manually installed qt to conan.

And the linkage fails because I use a wmain (and also a WinMain) and it does not find the main function.
What makes you believe regular qt behave like that?

Reply:
Because on "regular" qt, this file is linked by default https://github.com/qt/qtbase/blob/v5.15.2/src/winmain/qtmain_win.cpp